src/user.js

```diff
@@ -0,0 +1,44 @@
+import display from './ui.js';
+
+const refreshIcon = document.querySelector('.refresh');
+const addBtn = document.querySelector('.add-btn');
+const newTask = document.querySelector('.list-input');
+
+const addTask = (task) => {
+  let listData = JSON.parse(localStorage.getItem('tasks')) || [];
```

Comment:
- [ ] [optional] I noticed you always get your To-do lists data from localStorage each time you want to make use of it in the various modules, I would suggest you get it just once and export it when needed or use it as an argument in calling the function that needs the data, this way you don't repeat yourself.

Reply:
- [ ] I would also suggest that you create a module dir and put all your module files inside that dir for better project setup

Reply:
Thanks @Ezzywealth for your review, actually at the first I tried to export my array, but unfortunately because I assigned my array In UI it can't be const, but if I assign it let, I confront linters error. so this was the reason I defined it in each function